torch/nn/_functions/thnn/auto.py

```diff
@@ -3,13 +3,16 @@
 
 import torch
 from torch._thnn.utils import parse_header, THNN_H_PATH
-from torch.autograd.function import Function, InplaceFunction
+from torch.autograd import Variable
+from torch.autograd.function import Function, InplaceFunction, once_differentiable
 from torch._thnn import type2backend
+from .autoDoubleBackwards import double_backwards_fns
```

Comment:
please stick to the naming conventions we use everywhere else (`auto_double_backwards`)